src/assets/stylesheets/shared/_cards.less

```diff
@@ -20,6 +20,9 @@
       overflow-x: hidden;
       .list-group { padding-bottom: 0; }
       .list-view-pf-view { margin-top: -1px; }
+      &-lg {
+        max-height: 585px;
```

Comment:
The component looks better to me, thanks, but why in particular is it 585 px here?

Reply:
f8-card has a standard height of 440px and a -lg height of 650px, so given f8-card-body height of 375px I gave f8-card-body-lg 585px. +210px either way. *shrug*

Reply:
Ah makes sense, just wanted to confirm where the numbers came from, looks good to me.